starter-code/js/index.js

```diff
@@ -1,52 +1,87 @@
-function deleteItem(e){
+// function deleteItem(e){
 
-}
+// }
 
 function getPriceByProduct(itemNode){
 
-}
+  var itemQuantity = document.getElementsByTagName('input')[0]; //get user's input
+  var itemPrice = getElementsByClassName('price')[0];  //get the item's price
+  
+  
+  var priceByProduct = itemQuantity*itemPrice;  //calculate the total for the quantity of each item 
+  console.log(priceByProduct);
 
-function updatePriceByProduct(productPrice, index){
+  var itemTotals = [];
+  itemTotals.push(priceByProduct);
 
-}
+  return priceByProduct;
+  return itemTotals;
```

Comment:
you will not get to this return since this function already returns something prior to this.